CliFx.Tests/HelpTextSpecs.Commands.cs

```diff
@@ -82,6 +82,22 @@ private class RequiredOptionsCommand : ICommand
             public ValueTask ExecuteAsync(IConsole console) => default;
         }
 
+        [Command("cmd-with-enum-opts")]
+        private class EnumOptionsCommand : ICommand
+        {
+            public enum ValuesEnum { Value1, Value2, Value3 };
+
+            public enum EmptyEnum { };
```

Comment:
I would say we don't need to test for empty enum. I don't expect that to be a use case at all and even if it is, we can just assume it will work.

Reply:
Removed FTs for empty enums.